includes/Checker/Checks/Plugin_Readme_Check.php

```diff
@@ -198,12 +198,69 @@ private function check_stable_tag( Check_Result $result, string $readme_file, Pa
 	private function check_for_warnings( Check_Result $result, string $readme_file, Parser $parser ) {
 		$warnings = $parser->warnings ? $parser->warnings : array();
 
+		// This should be ERROR rather than WARNING. So ignoring here to handle separately.
+		if ( isset( $warnings['invalid_plugin_name_header'] ) ) {
+			unset( $warnings['invalid_plugin_name_header'] );
+		}
```

Comment:
Unset does not require an `isset()` check.

```suggestion
		unset( $warnings['invalid_plugin_name_header'] );
```